Jenkinsfile

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) YugaByte, Inc.
+
+properties([
+    parameters([
+        string(defaultValue: 'main', description: 'Specify the Branch name', name: 'BRANCH'),
+        booleanParam(defaultValue: false, description: 'If checked release builds are uploaded to s3 bucket. (debezium-connector -> s3://releases.yugabyte.com/debezium-connector-yugabytedb)', name: 'PUBLISH_TO_S3')
+    ])
+])
+
+pipeline {
+     agent {
+        node { label 'ybc-docker-agent' }
+    }
+    environment {
+        RELEASE_BUCKET_PATH = "s3://releases.yugabyte.com/debezium-connector-yugabytedb"
+    }
+    stages {
+        stage('Clean Workspace') {
+            steps {
+                cleanWs()
+            }
+        }
+        stage('Clone Project') {
+            steps {                  
+                git branch: '${BRANCH}', url: 'https://github.com/yugabyte/debezium-connector-yugabytedb.git'
+            }
+        }
+        stage('Build and Test') {
+            steps {
+                script{
+                    env.PKG_VERSION = sh(script: "mvn help:evaluate -Dexpression=project.version -q -DforceStdout", returnStdout: true).trim()
+                    env.ARTIFACT_ID = sh(script: "mvn help:evaluate -Dexpression=project.artifactId -q -DforceStdout", returnStdout: true).trim()
+                    sh 'sudo chmod 666 /var/run/docker.sock'
+                    sh 'mvn clean test package'
+                }
+            }
+        }
+        stage('Publish artifacts'){
+            steps {
+                script {
+                    if (env.PUBLISH_TO_S3) {
```

Comment:
this project also has to be released to a maven s3 repository and the Dockerfile has to be published to quay.io. 

Both these are already scripted through `man release`. Can this command be used instead? 
Actual commands are available here: https://github.com/yugabyte/debezium-connector-yugabytedb/blob/main/NOTES.md#release-process

Reply:
mvn release commands are needed only for release pipeline